examples/usecase-todo/keystone.ts

```diff
@@ -1,15 +1,14 @@
 import { config } from '@keystone-6/core'
-import { fixPrismaPath } from '../example-utils'
 import { lists } from './schema'
 import type { TypeInfo } from '.keystone/types'
 
 export default config<TypeInfo>({
   db: {
     provider: 'sqlite',
-    url: process.env.DATABASE_URL || 'file:./keystone-example.db',
+    url: process.env.DATABASE_URL || 'file:./keystone.example.db',
```

Comment:
```suggestion
    url: process.env.DATABASE_URL ?? 'file:./example.db',
```

Maybe this is a better name than `keystone.db`, and shorter too